plugins/evernote.koplugin/clip.lua

```diff
@@ -263,7 +264,17 @@ function MyClipping:parseHistoryFile(clippings, history_file, doc_file)
     end
     if lfs.attributes(doc_file, "mode") ~= "file" then return end
     local ok, stored = pcall(dofile, history_file)
-    if ok and stored.highlight then
+    if ok then
+        if not stored then
+            logger.warn("An empty history file ",
+                        history_file,
+                        "has been found. The book associated is ",
+                        doc_file)
+            return
+        end
```

Comment:
Shouldn't matter one way or the other, but I guess I've been trained to use elseif. :P